Orange/widgets/model/tests/test_owneuralnetwork.py

```diff
@@ -21,3 +21,18 @@ def test_migrate_setting(self):
         settings = dict(alpha=103)
         OWNNLearner.migrate_settings(settings, None)
```

Comment:
For simplicity purposes this could be changed to `OWNNLearner.migrate_settings(settings, 0)`. 
That's what actually happens, when there is no version.